trin-core/src/types/accumulator.rs

```diff
@@ -110,6 +108,10 @@ impl MasterAccumulator {
     /// preceding epoch's merkle root to historical epochs.
     // as defined in:
     // https://github.com/ethereum/portal-network-specs/blob/e807eb09d2859016e25b976f082735d3aceceb8e/history-network.md#the-header-accumulator
+    // Allow dead code since this functionality is no longer needed in the code base, now that we
+    // use a pre-merge, frozen master accumulator. However, somebody might find this code useful to
```

Comment:
I'm more in favor of removing this. We can always revive it from history if needed.

Reply:
Since we still use this multiple times in the testing, and I'd rather not remove those tests, I converted this functionality to a test utility function.